### ISSUE TITLE
Add artifact rejection module without normalization

### ISSUE THREAD

**Issue by timonmerk** [neuromodulation/py_neuromodulation#271]
Add feature-individual artifact rejection

**Comment by toni-neurosc** [comment]
Hi @timonmerk , just out of curiosity, I've seen that you added PARRM data afiltering a few months ago on the `nm_artifacts` module, but so far we don't have it integrated in the pipeline. I imagine this would be an additional `preprocessing` option in the settings? I also remember you mentioning this as an important feature in one of our conversations.

Before adding it we perhaps need to add tests and maybe an example of how it works? The example could also be a good test of whether it's working properly or not.

**Comment by timonmerk** [comment]
Yes, that's a good catch @toni-neurosc. I implemented the method, since it's very good at removing artifacts. But unfortunately the filter needs to be estimated beforehand, which doesn't make it real-time compatible.. That's why I didn't include it so far
